src/ObjectPool/src/DefaultObjectPoolProvider.cs

```diff
@@ -5,10 +5,17 @@
 
 namespace Microsoft.Extensions.ObjectPool
 {
+    /// <summary>
+    /// A provider of <see cref="DefaultObjectPool{T}"/>'s.
```

Comment:
Not super correct, this default implementation of `ObjectPoolProvider`. We don't guarantee that it returns `DefaultObjectPool`